webpack.config.js

```diff
@@ -0,0 +1,137 @@
+const path = require('path');
+const TerserJSPlugin = require('terser-webpack-plugin');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const OptimizeCSSAssetsPlugin = require('optimize-css-assets-webpack-plugin');
+const FixStyleOnlyEntriesPlugin = require('webpack-fix-style-only-entries');
+const ForkTsCheckerWebpackPlugin = require('fork-ts-checker-webpack-plugin');
+const { CleanWebpackPlugin } = require('clean-webpack-plugin');
+
+// Common plugins shared between storybook and production builds.
+const plugins = [
+  // Add Typescript type checking on build.
+  new ForkTsCheckerWebpackPlugin(),
+];
+
+// Common module rules shared between storybook and production builds.
+const rules = [
+  // Process all SCSS modules which will be compiled inside the main JS bundle.
+  {
+    test: /\.scss$/,
+    use: [
+      'style-loader',
+      {
+        loader: 'css-loader',
+        options: {
+          modules: true,
+          sourceMap: true,
+        },
+      },
+      'sass-loader',
+    ],
+    include: /\.module\.scss$/,
+  },
```

Comment:
This is the block that was duplicated between the two files. Both versions had to be modified to install the autoprefixer.